frontend/src/scenes/persons/Person.tsx

```diff
@@ -27,11 +27,11 @@ const { TabPane } = Tabs
 export const scene: SceneExport = {
     component: Person,
     logic: personsLogic,
-    paramsToProps: () => ({ syncWithUrl: true }),
+    paramsToProps: ({ params }) => ({ syncWithUrl: true, id: params._ }), // wildcard is stored in _
 }
 
-export function Person({ id: urlId }: { id?: string } = {}): JSX.Element {
-    const personsLogicProps: PersonLogicProps = { syncWithUrl: true }
+export function Person({ _: urlId }: { _?: string } = {}): JSX.Element {
+    const personsLogicProps: PersonLogicProps = { syncWithUrl: true, urlId }
```

Comment:
Why is this `urlId` here, but `id` in `paramsToProps` above? They are both creating props for the logic.

Reply:
whoops, nice catch! Relic of past, both should be `urlId`